gsa/src/web/pages/reports/detailspage.js

```diff
@@ -89,7 +89,9 @@ import FilterDialog from './detailsfilterdialog';
 
 const log = logger.getLogger('web.pages.report.detailspage');
 
-const DEFAULT_FILTER = Filter.fromString('rows=100');
+const DEFAULT_FILTER = Filter.fromString(
+  'levels=hm rows=100 min_qod=70 first=1 sort-reverse=severity',
```

Comment:
```suggestion
  'levels=hml rows=100 min_qod=70 first=1 sort-reverse=severity',
```
I would refrain from removing valid results at the moment. Even though they have low severity, they are existent vulnerabilities nonetheless.